uber/templates/fields/attendee.html

```diff
@@ -135,9 +135,11 @@ <h2 class="text-center">Dealer Application</h2>
             badge_type: '{{ c.ATTENDEE_BADGE }}',
 
             price: {{ badge_cost if badge_cost is defined else c.BADGE_PRICE }},
+            {% if c.BADGE_TYPE_PRICES and (attendee.is_new or attendee.badge_type == c.ATTENDEE_BADGE) %}
```

Comment:
This is the fix for https://github.com/magfest/ubersystem/issues/3472 -- we were setting anyone back to an attendee badge type if the "Attendee" badge button was clicked, which we _also_ do automatically when loading a form for someone without any kick-in levels. The only reason we need to change badge type is for events who use Attendee-facing badge types of different prices (configured via `c.BADGE_TYPE_PRICES`) so we hide this behind that flag, plus we don't do it anyway unless someone is new or an attendee to begin with.